code/analytics/src/main/java/com/adobe/marketing/mobile/analytics/internal/AnalyticsHitProcessor.kt

```diff
@@ -147,8 +145,10 @@ internal class AnalyticsHitProcessor(
                             AnalyticsConstants.EventDataKeys.Analytics.CONTENT_TYPE_HEADER
                         )
                     )
+                    val responseString = StreamUtils.readAsString(connection.inputStream)
                     val eventData: Map<String, Any?> = mapOf(
-                        AnalyticsConstants.EventDataKeys.Analytics.ANALYTICS_SERVER_RESPONSE to httpHeaders,
+                        AnalyticsConstants.EventDataKeys.Analytics.ANALYTICS_SERVER_RESPONSE to responseString,
+                        AnalyticsConstants.EventDataKeys.Analytics.HEADERS_RESPONSE to httpHeaders,
                         AnalyticsConstants.EventDataKeys.Analytics.HIT_HOST to url,
                         AnalyticsConstants.EventDataKeys.Analytics.HIT_URL to payload,
```

Comment:
can you also update host and url to the correct values?

Reply:
I verified this with our internal project.

Reply:
Yes, confusing, but the iOS implementation is the same: https://github.com/adobe/aepsdk-analytics-ios/blob/main/AEPAnalytics/Sources/AnalyticsHitProcessor.swift#L144